Tabby/components/FooterNavBar.tsx

```diff
@@ -1,61 +1,77 @@
 import { View, Pressable, Text } from "react-native";
 import { Link, usePathname } from "expo-router";
-import React from "react";
-import CameraButton from "@/components/CameraButton";
+import React, { useState } from "react";
 import SelectedLibrary from "@/assets/navbar-images/selectedLibrary.svg";
 import NotSelectedLibrary from "@/assets/navbar-images/notSelectedLibrary.svg";
 import SelectedExplore from "@/assets/navbar-images/selectedExplore.svg";
 import NotSelectedExplore from "@/assets/navbar-images/notSelectedExplore.svg";
 import SelectedFavorite from "@/assets/navbar-images/selectedFavorite.svg";
 import NotSelectedFavorite from "@/assets/navbar-images/notSelectedFavorite.svg";
 import Settings from "@/assets/navbar-images/settings.svg";
+import CameraIcon from "@/assets/navbar-images/camera.svg";
+import CameraModal from "@/components/camera/CameraModel";
 
 const FooterNavBar = () => {
-  // getting the current page to see the path and using that to make the icons change to selected if that path invloves the page that icon is related to 
+  // set to true to show camera modal
+  const [isCameraModalVisible, setCameraModalVisible] = useState(false);

```

Comment:
> set to true to show camera modal

I feel like this code is doing the opposite of what the comment says. Can this be checked?